### PR TITLE
Add guideline for ignored values

### PR DESCRIPTION
**Pull request by vladimir-v-diaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -650,7 +650,24 @@ underscore if a separation is needed, such as: popanitem or pop_an_item
...
+### Ignored Values ###
+
+The variable name `junk` should be used in cases where a returned valued must be
+stored but the variable itself is unneeded.
+
+  No:
+
+```Python
+for _ in range(10):     
+  do_something()
```
Comment (awwad) [inline README.md]: Would you say that `i` is permissible for an unused iterator? This example might imply to people that it's not.

**Comment by awwad** [comment]
This is for pylint rules, I guess? Seems like a sensible recommendation, though I think I might add a few words on why. Something like "In order to make clear to the reader that a variable is not used...", or even just "for readability".

**Comment by vladimir-v-diaz** [comment]
> Would you say that i is permissible for an unused iterator?

By convention, `i,` `j,` `k,` and `_,` are often used in this case.  Although Pylint follows the i, j, k, _ convention, Justin recommended we use  `junk` in instances where a returned value is unused.

In the guideline, I say: "The variable name `junk` should be used in cases where a returned valued must be stored but the variable itself is unneeded."  Is this not clear enough?  Should I justify why we recommend `junk` over the i, j, k, _ convention?

**Comment by awwad** [comment]
I think an explanation would probably be good for why this guideline exists, yes.

I think that using "junk" where a returned value is unused makes sense, yes (example 2). I'm not sure I agree on the iterators (example 1), though.

**Comment by vladimir-v-diaz** [comment]
@JustinCappos Are you okay with allowing `i,` `j,` or `k` for iterators?

**Comment by JustinCappos** [comment]
Yes, this is fine by me.

**Comment by vladimir-v-diaz** [comment]
Okay, I added more examples and an explanation.  Let me know if anything else is needed.

**Comment by awwad** [comment]
👍  LGTM

**Pull request merged**